src/javascript-node-translation-info-factory.js

```diff
@@ -0,0 +1,35 @@
+const Pofile                = require('pofile');
+const { MARKER_NO_CONTEXT } = require('./constants.js');
+
+function toString(withLineNumbers = false) {
+  return (withLineNumbers && this.line)
+    ? `${ this.file }:${ this.line }`
+    : this.file;
+}
+
+function toPoItem(withLineNumbers = false) {
+  let poItem = new Pofile.Item();
+
+  poItem.msgid      = this.text;
+  poItem.references = [ this.reference.toString(withLineNumbers) ];
+  poItem.msgstr     = [];
```

Comment:
same here ;) please be consistent with the current codestyle, maybe ? did you eslint-check your PR ? we do have an .eslintrc !

Reply:
Yep, I have ESLint checking the code in background, but It seems the config does not have such a rule to prevent this code format